internal/context/init.go

```diff
@@ -171,42 +171,87 @@ func getProjectPrompts(projectName string, modules map[string]moduleconfig.Modul
 	return handlers
```

Comment:
Mind renaming context while you're in there? I'd like to get rid of that. I think this should probably be renamed to `internal/init` to match what we've been going for with other commands where the package is named after the command in `cmd`.